src/components/PlaylistMenuBtn/PlaylistMenuBtn.jsx

```diff
@@ -0,0 +1,15 @@
+import { useVideoContext } from "../../context/videoContext";
+import {FaTrash} from "../../icons/icons";
+import { useParams } from "react-router-dom";
+import { removeVideoFromPlaylistCall } from "../../utils/utils";
+import { useAuthContext } from "../../context/AuthContext";
+export function PlaylistMenuBtn({videoId}){
+    const {dispatch}=useVideoContext();
+    const {userState}=useAuthContext();
+    const {playlistId}=useParams();
+    return (
```

Comment:
You can give a line space here. Before `return`.

Reply:
done